src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -817,9 +817,10 @@ TEST_CASE("request_rate_sequence")
 
 /// Check that the inference requests are balanced across sequences
 ///
-TEST_CASE("request rate sequence: verify inferences across sequences")
+TEST_CASE("request rate sequence: verify behavior of serial_sequences option")
```

Comment:
call the test serial sequences, not request rate sequence

Reply:
except this tests multiple things. It doesn't just verify the behavior of serial sequences. Also, the comment on 818 isn't congruent with the test name.

Reply:
updated name to serial sequences